ckanext/dia/model.py

```diff
@@ -0,0 +1,125 @@
+# encoding: utf-8
+
+from __future__ import print_function
+import datetime
+import logging
+import sys
+import six
+from sqlalchemy import Table, Column, types, ForeignKey
+from urllib.parse import urljoin, quote
+from os.path import join
+import uuid
+
+from ckan import model
+from ckan.lib.navl.dictization_functions import validate
+from ckan.lib.navl.validators import not_empty
+from ckan.logic import ValidationError
+from ckan.logic.validators import Invalid
+from ckan.model import DomainObject, User
+from ckan.model.meta import metadata, mapper
+from ckan.plugins import toolkit
+from ckan.common import _, config
+
+
+log = logging.getLogger(__name__)
+minted_uri_table = None
+
+
+def db_setup():
+    if minted_uri_table is None:
+        define_table()
+
+    if not model.package_table.exists():
+        log.critical("Exiting: can not migrate minted uri model \
+if the database does not exist yet")
```

Comment:
Python has implicit string concatenation so you can do this like: 
```suggestion
        log.critical("Exiting: can not migrate minted uri model"
                           "if the database does not exist yet")
```